client/app/account/user/advanced/user-advanced.html

```diff
@@ -1,83 +1,86 @@
                             <input type="checkbox"
                                    id="ovhMagSubscription"
                                    class="oui-switch oui-switch__checkbox"
-                                   data-ng-model="$ctrl.developmentMode.enabled"
-                                   data-ng-change="$ctrl.updateDevelopmentMode()"
-                                   data-ng-disabled="$ctrl.isLoadingDeveloperMode" />
+                                   data-ng-model="$ctrl.developerModeRestriction.enabled"
+                                   data-ng-change="$ctrl.updateDeveloperModeRestriction()"
+                                   data-ng-disabled="$ctrl.isUpdating" />
                             <div class="oui-switch__slider"></div>
                             <span class="oui-icon oui-icon-success" aria-hidden="true"></span>
                             <span class="oui-icon oui-icon-error" aria-hidden="true"></span>
                         </label>
-                        <span data-translate="user_account_advanced_section_developer_label"></span>
-                    </div>
-
-                    <h3 class="mt-5"
-                        data-translate="user_account_advanced_section_developer_join_us">
-                    </h3>
-
-                    <ul class="follow-link-list list-unstyled my-5">
-                        <li class="mb-2">
-                            <span class="fa fa-2x fa-github text-center mr-2"
-                                  aria-hidden="true">
-                            </span>
-                            <a target="_blank"
-                               href="https://github.com/ovh-ux"
-                               class="align-bottom">
-                                https://github.com/ovh-ux
-                            </a>
-                        </li>
-                        <li class="mb-2">
-                            <span class="fa fa-2x fa-comments text-center mr-2"
-                                  aria-hidden="true">
-                            </span>
-                            <a target="_blank"
-                               href="https://gitter.im/ovh/ux"
-                               class="align-bottom">
-                                https://gitter.im/ovh/ux
-                            </a>
-                        </li>
-                    </ul>
-                </form>
-
+                        <span data-ng-if="!$ctrl.isLoading"
+                              data-translate="user_account_advanced_section_developer_label">
+                        </span>
+                    </form>
+                </div>
+                <h3 class="mt-5"
+                    data-translate="user_account_advanced_section_developer_join_us">
+                </h3>
+                <ul class="list-unstyled my-5">
+                    <li class="mb-2">
+                        <span class="fa fa-2x fa-github text-center mr-2 align-middle"
```

Comment:
This `<span>` should be in the `<a>` tag, right?

If you click on the icon, I think the link should work :)